.eslintrc.cjs

```diff
@@ -145,9 +145,10 @@ const rules = {
       'allowAllPropertiesOnSameLine': true,
     },
   ],
-  'operator-linebreak': 'error',
+  'operator-linebreak': 'off',
   'prefer-arrow-callback': 'error',
   'prefer-const': 'error',
+  'prefer-promise-reject-errors': 'off',
```

Comment:
Sorry, but could you explain the eslint changes to me too? I'm not sure I understand why these need to change here.

Reply:
I don't know eighrt, I just upgrade eslint and plugins version then these errors are reported

Reply:
I guess they are actually broken before the upgrade

Reply:
Oh because `eslint-config-google` change both these rule's option

Reply:
It's not clear to me why eslint and eslint-config-google needed to be updated? Did the old version of eslint not have node18 support?

Reply:
They do, I just upgrade eslint and update its plugin by the way

Reply:
Oh, I guess that's my confusion then, because I was trying to understand why there were eslint changes in the same PR as a tsconfig target patch if they weren't related.